openlibrary/plugins/openlibrary/lists.py

```diff
@@ -353,6 +353,23 @@ def POST(self, user_key: str | None, list_key: str | None = None):  # type: igno
             return safe_seeother(list_record.key)
 
 
+class lists_add_account(delegate.page):
+    path = r"/account/lists/add.*"
+
+    def GET(self):
+        if user := get_current_user():
+            i = web.input(name=None)
+            return web.seeother(
+                f'{user.key}/lists/add{ "?seeds=" + i.seeds if i.seeds else ""}'
```

Comment:
We might be able to use `web.ctx.query` to get the `?seeds=...` 
```suggestion
                f'{user.key}/lists/add{web.ctx.query}'
```

See https://webpy.org/cookbook/ctx